actors/builtin/miner/miner_test.go

```diff
@@ -4640,6 +4653,70 @@ func (h *actorHarness) preCommitSector(rt *mock.Runtime, params *miner.PreCommit
 	return h.getPreCommit(rt, params.SectorNumber)
 }
 
+type preCommitBatchConf struct {
+	sectorWeights []market.SectorWeights
+}
+
+func (h *actorHarness) preCommitSectorBatch(rt *mock.Runtime, params *miner.PreCommitSectorBatchParams, conf preCommitBatchConf, first bool) []*miner.SectorPreCommitOnChainInfo {
+	rt.SetCaller(h.worker, builtin.AccountActorCodeID)
+	rt.ExpectValidateCallerAddr(append(h.controlAddrs, h.owner, h.worker)...)
+	{
+		expectQueryNetworkInfo(rt, h)
+	}
+	sectorDeals := make([]market.SectorDeals, len(params.Sectors))
+	sectorWeights := make([]market.SectorWeights, len(params.Sectors))
+	anyDeals := false
+	for i, sector := range params.Sectors {
+		sectorDeals[i] = market.SectorDeals{
+			SectorExpiry: sector.Expiration,
+			DealIDs:      sector.DealIDs,
+		}
+
+		if len(conf.sectorWeights) > i {
+			sectorWeights[i] = conf.sectorWeights[i]
+		} else {
+			sectorWeights[i] = market.SectorWeights{
```

Comment:
suggestion: it would be nice to comment on the `sectorWeights` fields of the preCommitBatchConf that these weights are used for sectors 0 through `len(sectorWeights)` and not all need to be used.